README.md

```diff
@@ -32,7 +32,7 @@ regarding Chainlink social accounts, news, and networking.
 
 ## Build Chainlink
 
-1. [Install Go 1.21.1](https://golang.org/doc/install), and add your GOPATH's [bin directory to your PATH](https://golang.org/doc/code.html#GOPATH)
+1. [Install Go 1.21.3](https://golang.org/doc/install), and add your GOPATH's [bin directory to your PATH](https://golang.org/doc/code.html#GOPATH)
```

Comment:
WDYT about using `1.21` or `1.21.x`, so that we don't have to update for every patch release?
```suggestion
1. [Install Go 1.21](https://golang.org/doc/install), and add your GOPATH's [bin directory to your PATH](https://golang.org/doc/code.html#GOPATH)
```

Reply:
Makes sense! There are actually more reasons to go for it, e.g. [.tool-versions](https://github.com/smartcontractkit/chainlink/blob/c3b156b6bd202870eaa6ae037b25aaada11c6852/.tool-versions#L1) mentioning 1.21.5..